QuizWhois.Domain/Services/Implementations/QuestionService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using QuizWhois.Common.Models;
+using QuizWhois.Domain.Database;
+using QuizWhois.Domain.Entity;
+using QuizWhois.Domain.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuizWhois.Domain.Services.Implementations
+{
+    public class QuestionService : IQuestionService
+    {
+        public ApplicationContext _context { get; set; }
+
+        public QuestionService(ApplicationContext context)
+        {
+            _context = context;
+        }
+
+        public void AddQuestion(QuestionModel operationModel)
```

Comment:
It is better to return any kind of result to the caller to notify whether operation was successful or not. Return the same object, now with Id, or at least boolean